docs/user/filesystem/partition_schemes.rst

```diff
@@ -0,0 +1,128 @@
+.. _fsds_partition_schemes:
+
+Partition Schemes
+=================
+
+Partition Schemes can be defined using a JSON/typesafe configuration or by a common name. By default these schemes all
+utilize "leaf storage" with a one up sequence number. This allows for appending data when it arrives out of order.
+
+Common Date schemes are:
+
+* minute
+* hourly
+* daily
+* monthly
+* julian-minute
+* julian-hourly
+* julian-daily
```

Comment:
What's the difference between minute (etc) and julian-minute?  Also, I wonder if we should only mention daily and monthly.  

Oh, is weekly possible?  (E.g., weeks since the epoch?)